tests/test_post_to_studio.py

```diff
@@ -132,15 +132,24 @@ def test_post_to_studio_failed_start_request(
     assert mocked_post.call_count == 1
 
 
-def test_post_to_studio_end_only_once(tmp_dir, mocked_dvc_repo, mocked_studio_post):
+def test_post_to_studio_done_only_once(tmp_dir, mocked_dvc_repo, mocked_studio_post):
     mocked_post, _ = mocked_studio_post
     with Live() as live:
         live.log_metric("foo", 1)
         live.next_step()
 
-    assert mocked_post.call_count == 4
+    expected_done_calls = [
+        call
+        for call in mocked_post.call_args_list
+        if call.kwargs["json"]["type"] == "done"
+    ]
     live.end()
-    assert mocked_post.call_count == 4
+    actual_done_calls = [
+        call
+        for call in mocked_post.call_args_list
+        if call.kwargs["json"]["type"] == "done"
+    ]
+    assert expected_done_calls == actual_done_calls
```

Comment:
Better to test that this doesn't call `done` again rather than test call counts.